### PR TITLE
Validated inputs in the form and add disabled button submit if one of the inputs is invalid

### PR DESCRIPTION
**Pull request by Psaruklily**

### CODE REVIEW

File: src/app/form/form.component.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                    <input type="email" class="form-control" name="email" formControlName="studentEmail">
+                    <div class="warning" *ngIf="currentStudent.controls['studentEmail'].invalid && currentStudent.controls['studentEmail'].touched">
+                        Enter a valid email address. (eg: yourname@domain.com)
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <label class="label">
+                <span>University / College of attendance</span>
+              </label>
```
Comment (Oleh1033) [inline src/app/form/form.component.html:91]: formatting

File: src/app/form/form.component.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                    <input type="text">
+                </div>
+
+                <div class="form-group">
+                    <label class="label">
+              <span>Your Message</span>
+            </label>
```
Comment (Oleh1033) [inline src/app/form/form.component.html:98]: look above

File: src/app/form/form.component.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                </div>
+
+                <div class="btn-submit">
+                    <input type="submit" value="Submit" [disabled]="currentStudent.invalid" [ngClass]="{'invalidForm': isInvalidForm}" *ngIf="currentStudent.invalid; else unset">
+                    <ng-template #unset>
+                        <input type="submit" value="Submit" [ngClass]="{'validForm': isValidForm}">
+                    </ng-template>
+                </div>
+            </form>
+            <p class="spacesBelowForm">-----------------------------------------------------------------------------------------------------------------------------</p>
```
Comment (Oleh1033) [inline src/app/form/form.component.html]: what is this?

**Review by Oleh1033 — CHANGES_REQUESTED** [review]
some small changes required

File: src/app/form/form.component.html (hunk 3 above)
Comment (Psaruklily) [reply src/app/form/form.component.html]: image

Comment (Psaruklily) [reply src/app/form/form.component.html]: It's a line under button

File: src/app/form/form.component.html (hunk 1 above)
Comment (Psaruklily) [reply src/app/form/form.component.html:91]: I can't to align this label. When I'm selecting a part of the code which I wish to align, and I'm pressing shortcut Shift+Alt+F (for align html code on Windows), nothing changes.

File: src/app/form/form.component.html (hunk 2 above)
Comment (Psaruklily) [reply src/app/form/form.component.html:98]: The same story

File: src/app/form/form.component.html (hunk 3 above)
Comment (Oleh1033) [reply src/app/form/form.component.html]: ok

File: src/app/form/form.component.html (hunk 1 above)
Comment (Oleh1033) [reply src/app/form/form.component.html:91]: align this code by your hand
why <span> bellow <label>

**Pull request merged**